Algorithms/Common/DayofWeek.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+
+namespace Algorithms.Common
+{
+    public static class DayofWeek
+
+    {
+        /// <summary>
+        /// Determine day of week of the date
+        /// </summary>
+        /// <returns>Return string of day of week of the date</returns>
+        /// <param name="date">Date.</param>
+        public static string GetDayofWeek(DateTime date)
+        {
+
+            int count = 1; //Int to shows Ordinal number of occurrence of day of week
+            DateTime Occurrence = date.AddDays(-7); //Date to check the ordinal number of day of week 
+
+            for (string month = date.ToString("MMMM"); month == Occurrence.ToString("MMMM"); Occurrence = Occurrence.AddDays(-7))
+            {
+                count++;
+            }
+
+            //switch case for count to return string of day of week of the date
+            switch (count)
+            {
+                case 1:
+                    return count + "st " + date.ToString("ddd");
+                case 2:
+                    return count + "nd " + date.ToString("ddd");
+                case 3:
+                    return count + "rd " + date.ToString("ddd");
+                default:
+                    return count + "th " + date.ToString("ddd");
+            }
+
+        }
+
+
+        /// <summary>
+        /// Determine whether the day of week  
+        /// </summary>
+        /// <returns>Return string of day of week of the date</returns>
+        /// <param name="date">Date.</param>
+        /// <param name="month">Month.</param>
+        /// <param name="year">Year.</param>
+        //overload
+        public static string GetDayofWeek(int date, int month, int year)
+        {
+            DateTime date2 = new DateTime(year, month, date);
+            return GetDayofWeek(date2);
```

Comment:
1. not _date_, but _day_
2. Easy return:
`return GetDayofWeek(new DateTime(year, month, day));`